main.go

```diff
@@ -37,53 +121,41 @@ func main() {
 }
 
 func RollingBackupCommand() cli.Command {
+
+	snapshotFlags := []cli.Flag{
+		cli.DurationFlag{
+			Name:  "creation",
+			Usage: "Create backups after this time interval in minutes",
+			Value: 5 * time.Minute,
+		},
+		cli.DurationFlag{
+			Name:  "retention",
+			Usage: "Retain backups within this time interval in hours",
+			Value: 24 * time.Hour,
+		},
+		cli.BoolFlag{
+			Name:  "once",
+			Usage: "Take backup only once",
+		},
+	}
+
+	snapshotFlags = append(snapshotFlags, commonFlags...)
+
 	return cli.Command{
-		Name:   "rolling-backup",
-		Usage:  "Perform rolling backups",
-		Action: RollingBackupAction,
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:  "endpoints",
-				Usage: "Etcd endpoints",
-				Value: "127.0.0.1:2379",
-			},
-			cli.DurationFlag{
-				Name:  "creation",
-				Usage: "Create backups after this time interval in minutes",
-				Value: 5 * time.Minute,
-			},
-			cli.DurationFlag{
-				Name:  "retention",
-				Usage: "Retain backups within this time interval in hours",
-				Value: 24 * time.Hour,
-			},
-			cli.BoolFlag{
-				Name:   "debug",
-				Usage:  "Verbose logging information for debugging purposes",
-				EnvVar: "RANCHER_DEBUG",
-			},
-			cli.BoolFlag{
-				Name:  "once",
-				Usage: "Take backup only once",
+		Name:  "rke-tools",
+		Usage: "Perform rke tools",
```

Comment:
I am not sure i understand the name here, can we at least add more descriptive usage message for the whole tool, this should be a tool to perform backup operations on etcd